backend/src/main/java/ai/verta/modeldb/experimentRun/FutureExperimentRunDAO.java

```diff
@@ -44,16 +44,20 @@
   private final DatasetHandler datasetHandler;
   private final PredicatesHandler predicatesHandler;
   private final SortingHandler sortingHandler;
+  private final HyperparametersFromConfigHandler hyperparametersFromConfigHandler;
+  private final boolean populateConnectionsBasedOnPrivileges;
 
   public FutureExperimentRunDAO(
       Executor executor,
       FutureJdbi jdbi,
       UAC uac,
       ArtifactStoreDAO artifactStoreDAO,
-      DatasetVersionDAO datasetVersionDAO) {
+      DatasetVersionDAO datasetVersionDAO,
+      boolean populateConnectionsBasedOnPrivileges) {
```

Comment:
I'd recommend passing the whole config here as we'll most likely need other fields later and we don't want to add them as more constructor parameters.

Reply:
Added config instead of individual flag